lib/Limitation/Mapper/LocationLimitationMapper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Limitation\Mapper;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use EzSystems\RepositoryForms\Limitation\LimitationFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+class LocationLimitationMapper implements LimitationFormMapperInterface
+{
+    /**
+     * Form template to use.
+     *
+     * @var string
+     */
+    private $template;
+
+    public function mapLimitationForm(FormInterface $form, Limitation $data)
+    {
+        $form->add('limitationValues', 'hidden', [
+            'data' => $data->limitationValues[0],
+            'required' => false,
+            'label' => $data->getIdentifier(),
+        ]);
+    }
+
+    public function setFormTemplate($template)
+    {
+        $this->template = $template;
+    }
+
+    public function getFormTemplate()
+    {
+        return $this->template;
+    }
+
+    public function filterLimitationValues(Limitation $limitation)
+    {
+        $limitation->limitationValues = [$limitation->limitationValues];
```

Comment:
Possibly, but it works... The form returns the value directly, but the limitation wants it in an array. So I wrap it here, and unwrap it in `mapLimitationForm()`.